mpact_api/mpact/constants.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+class Constants:
```

Comment:
No need to stick these in a class. They are in their own module, so they already have a namespace.